src/utils/populateInstrumentsUtils.js

```diff
@@ -0,0 +1,20 @@
+import { fetchApiCall } from './api.js'
+import { useConfigurationStore } from '../stores/configuration.js'
+
+export const getFilterList = async () => {
+  const configurationStore = useConfigurationStore()
+  const response = await fetchApiCall({
+    url: `${configurationStore.observationPortalUrl}instruments`,
+    method: 'GET'
+  })
+  const filterList = []
+  Object.values(response).forEach((instrument) => {
+    if (instrument.class === '0m4' && instrument.optical_elements.filters) {
```

Comment:
this is probably fine for now, but if Wayne's telescopes are integrated they may be 0m35 or something different, and we generally would want to keep this easily adjustable. You could put it in a global at the top of the file, or read it from the public/config/config.json which we can set in the deployment at runtime!

Reply:
oo I like that!

Reply:
I'm intrigued by this `read it from the public/config/config.json which we can set in the deployment at runtime`. I'll be declaring it as a global config and ask you about that statement in person (or you can also explain here but I'm patient and willing to rewrite this when the time comes)